app/config/README.md

```diff
@@ -2,15 +2,15 @@
 
 This folder contains the default configuration files for the jitsi-party application.
 Both the server and the client use these files.
-Do not alter them directly.
 
-To configure your custom installation of jitsi-party, drop new files into the `overrides` folder here.
+`base` contains the basic/default configurations, on which oyur custom configs will build. Do not alter these files.
+
+To configure your custom installation of jitsi-party, symlink your configuration files into the `overrides` folder.
+The contents of this folder are gitignored, so the symlinks will not be stored in the repository.
+By convention, a party name `my-party` will keep its configs in `config/my-party/`, and you can symlink `config/my-party/*` into `config/overrides` when you are ready to run your party server.
```

Comment:
I know the setup process to symlink is already fairly simple, but I wonder if it's worth adding a script that just takes the party name as a param. So `setup_configs my-party` clears out the existing overrides and symlinks in `config/my-party`.

Reply:
down to do this as a separate change